summary.Rmd

```diff
@@ -2,12 +2,40 @@ Summary
 =======
 
 1. Use available tools
+    a. Image registries (e.g. Docker Hub),
+    b. Dockerfile generators (e.g. repo2docker)
+    c. Project templates (e.g. cookie cutter) could go here and one Dockerfile per project?
+    d. Linters
 2. Use versioned images
+    a. Mention specific sha digests?
 3. Format for clarity
+    a. Use indentation, line breaks and unabbreviated arguments
+    b. Clarity > brevity
+    c. Group related commands
 4. Document within the Dockerfile
-5. Order the instructions
+    a. Add comments explaining the steps
+    b. Use labels for metadata
+    c. Include build and run commands
 6. Specify software versions
+    a. Pin system library versions
+    b. Explictly list package versions to install
+    c. Comment why each is needed
+9. Use version control
+    a. Include CI testing
+    b. Use one Dockerfile per project
+    c. Use project templates (e.g. cookie cutter)
```

Comment:
This seems redundant too. Are we discussing it twice in the paper (maybe we should consolidate, or tie to one section and only briefly mention in the other, if needed?)

Reply:
Yes, it is redundant - I added it to Rule 1 as a suggestion, where I think it fits more naturally, os would advocate removing it from this rule.

Reply:
I do think the project templates belongs better with rule 1, if you think of a template as a kind of tool. I'm still not sure of the one Dockerfile per project (see discussion in https://github.com/nuest/ten-simple-rules-dockerfiles/pull/65#discussion_r420239648).

Reply:
So we both agree to move templating to Rule 1 and remove it from Rule 9? As noted above, the "one Dockerfile per project" clause can then also be dropped in favour of your more general statement.

Reply:
Agree on both parts!

Reply:
Done!